ui/src/shared/components/lastRunTaskStatus/LastRunTaskStatus.tsx

```diff
@@ -0,0 +1,80 @@
+// Libraries
+import React, {FC, useRef, useState} from 'react'
+import classnames from 'classnames'
+
+// Components
+import {
+  Icon,
+  IconFont,
+  Popover,
+  Appearance,
+  PopoverInteraction,
+  ComponentColor,
+} from '@influxdata/clockface'
+
+// Styles
+import './LastRunTaskStatus.scss'
+
+interface PassedProps {
+  lastRunError?: string
+  lastRunStatus: string
+}
+
+const LastRunTaskStatus: FC<PassedProps> = ({lastRunError, lastRunStatus}) => {
+  const triggerRef = useRef<HTMLDivElement>(null)
+  const [highlight, setHighlight] = useState<boolean>(false)
+
+  let color = ComponentColor.Success
```

Comment:
are there implications to setting the default state to `success`? if a task run fails and we don't know about it, and we set the status to success, there might be a nonzero impact on the user? what kind of thought have we put into that?

Reply:
If a task run fails and we don't know about it, that's a failure on the API